pkg/commands/serviceversion/deactivate.go

```diff
@@ -48,7 +49,7 @@ func NewDeactivateCommand(parent argparser.Registerer, g *global.Data) *Deactiva
 // Exec invokes the application logic for the command.
 func (c *DeactivateCommand) Exec(_ io.Reader, out io.Writer) error {
 	serviceID, serviceVersion, err := argparser.ServiceDetails(argparser.ServiceDetailsOpts{
-		AllowActiveLocked:  true,
+		Active:             optional.Of(true),
```

Comment:
This is one of the actual bug fix changes: `fastly service-version deactivate` will now fail of the specified service-version is not active.